### DISCUSSION TITLE
Allow tracing atom value changes with useAtomDevtools

### DISCUSSION THREAD

**Discussion by reuben** [pmndrs/jotai#1534] [Ideas]
Redux DevTools support tracing so you can get a stack trace for state changes: https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#trace

It would be nice to similarly be able to do `useAtomDevtools(atom, { trace: true })` to get that functionality for atom changes.

**Comment by dai-shi** [comment]
Is this the same "trace" as https://github.com/pmndrs/valtio/issues/331 in another repo?
(Then, https://github.com/pmndrs/zustand/issues/716 is also related.)

cc: @Aslemammad

**Comment by Aslemammad** [comment]
Yep, It is related, and I do not know when it can be fixed really.

**Reply by dai-shi** [reply]
Oops, I misread the OP. Thought it's something recently implemented.
